DownFall/Views/MenuSpriteNode.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Menu.swift
+//  DownFall
+//
+//  Created by William Katz on 3/10/19.
+//  Copyright © 2019 William Katz LLC. All rights reserved.
+//
+
+import SpriteKit
+
+class MenuSpriteNode: SKSpriteNode {
+    
+    struct Constants {
+        static let pauseMenu = "pauseMenu"
+        static let resume = "resume"
+    }
+    
+    enum MenuType {
+        case pause
+        
+        func buttonText() -> [String] {
+            switch self {
+            case .pause:
+                return ["Resume"]
+            }
+        }
+    }
+    
+    init(_ menuType: MenuType, playableRect: CGRect) {
+        let menuSizeWidth = playableRect.size.width * 0.5
+        let menuSizeHeight = playableRect.size.height * 0.5
+        
+        
+        super.init(texture: SKTexture(imageNamed: "menu"), color: .black, size: CGSize(width: menuSizeWidth, height: menuSizeHeight))
+        isUserInteractionEnabled = true
+//        super.init()
+        self.name = Constants.pauseMenu
+        
+//        let buttonTexts = menuType.buttonText()
```

Comment:
This was a remnant of trying to make this class reusable.  It is a worthwhile endeavor but does not belong in this PR.  Create a TODO and link a trello ticket